guides/source/active_job_basics.md

```diff
@@ -127,6 +127,18 @@ end
 Queues
 ------
 
+The default queue jobs are added to is named `:default`. You can change the 
+default queue using `config.active_job.default_queue_name` in `application.rb`:
```

Comment:
Thanks @shunsukeaida - I think this PR is not going to be merged. You might be interested in reviewing this related PR: #19712